build/ci-tf-bin.Dockerfile

```diff
@@ -6,13 +6,13 @@ RUN microdnf update -y && \
     microdnf install -y git make go-toolset && \
     microdnf clean all
 
-# terraform-provider-ocm repo
-COPY . ./terraform-provider-ocm
+# terraform-provider-red-hat-cloud-services repo
+COPY . ./terraform-provider-red-hat-cloud-services
```

Comment:
@enriquebelarte we also need a PR for openshift/release repo, doesn't we?

Reply:
Updated this to `rhcs` as well...

We have a PR  https://github.com/openshift/release/pull/40677